circuit_maintenance_parser/providers.py

```diff
@@ -68,7 +68,9 @@ def process(self, data_type: str = "") -> Iterable[Maintenance]:
         for parser_class in self._parser_classes:
             try:
                 parser = parser_class(
-                    raw=self.raw, provider_type=self.get_provider_type(), default_organizer=self._default_organizer,
+                    raw=self.raw,
+                    default_provider=self.get_provider_type(),
+                    default_organizer=self.get_default_organizer(),
```

Comment:
Do I understand correctly that `provider_type` is not actually an attribute on the parser class, and so it was being ignored altogether? If so, we might consider configuring `extra = 'forbid'` on our Pydantic classes (https://pydantic-docs.helpmanual.io/usage/model_config/) so that mis-specified attributes cause an error rather than being silently ignored, so as to more readily catch issues like this in the future.

Reply:
that would be great!